sdk/cosmos/azure-cosmos/azure/cosmos/_models.py

```diff
@@ -0,0 +1,54 @@
+# The MIT License (MIT)
+# Copyright (c) 2023 Microsoft Corporation
+
+from typing import Any
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+
+class BatchOperationResponse:
+    operation: dict
+    """Operation that was sent as part of the batch request."""
+    operation_response: dict
```

Comment:
I feel like we can just return this response directly. Remind me - do we guarantee that the response ordering == input ordering? If so, we don't need to include the input with the output.

Reply:
we do but in our other SDKs we also provide the pairing so I don't think this is a big deal